x-pack/legacy/plugins/canvas/canvas_plugin_src/renderers/time_filter/components/time_picker_mini/index.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export { TimePickerMini } from './time_picker_mini';
```

Comment:
nit: I've been renaming `mini` to `popover`, as that seems much more descriptive of the underlying component.

Reply:
taken care of. I had to do change a css file and a template as well but it should all be switched from TimePickerMini -> TimePickerPopover